src/devtools/views/Components/Tree.js

```diff
@@ -113,7 +113,7 @@ export default function Tree(props: Props) {
             <FixedSizeList
               className={styles.List}
               height={height}
-              innerElementType={innerElementType}
+              innerElementType={TreeWrapper}
```

Comment:
nit: I kind of liked the `innerElementType` name better. `TreeWrapper` to me indiates something that wraps this whole component (since its name is `Tree`) rather than something that goes inside of the list.